.github/workflows/build.yml

```diff
@@ -1,126 +1,97 @@
 name: Build
-on: [push, pull_request]
+on:
```

Comment:
Would it make sense to run this when pushing to a tag to create releases?

https://futurestud.io/tutorials/github-actions-run-a-workflow-when-creating-a-tag#runworkflowfornewtags